pkg/collector/corechecks/containers/docker.go

```diff
@@ -175,6 +175,7 @@ func (d *DockerCheck) Run() error {
 	collectingContainerSizeDuringThisRun := d.instance.CollectContainerSize && d.collectContainerSizeCounter == 0
 
 	images := map[string]*containerPerImage{}
+	currentUnixTime := time.Now().Unix()
```

Comment:
same here, currentUnixTime is sampled before the loop and there are some potentially slow calls in the loop (i'm thinking of `GetStorageStats`, not sure if there are others)

Reply:
Fixed